schema/config-schema.json

```diff
@@ -118,6 +118,10 @@
                 "user": {
                     "id": "https://opencontainers.org/schema/bundle/process/user",
                     "type": "object",
+                    "required": [
+                        "uid",
+                        "gid"
```

Comment:
> should the user field should be removed from the `config-schema`, into different types (`config-linux.json`/`config-windows.json` etc).

You could do that if you like, but I don't think it's worth the trouble.  I'd rather leave the user fields optional in the JSON Schema and check for any platform-specific requirements in runtime-tools' Go validation.